docs/policy/storage_policy.md

```diff
@@ -0,0 +1,72 @@
+Interestingly, Many users did not even log into their instances of the hub during the past 6 - 12 months. 
+
+One of the user home directory snapshotted below had a database file named "fec_nyc.db" whose size was around ~900 GB of data. This database file was used as part of the Data 100 course lab 4 where students learn to work with a database using the SQLAlchemy package. 
+
+![alt text](user2_snapshot.png "Single database file whose size is 900 GB")
+
+Another user had a core.rsession file whose size was around 785 GB. She had this file under the course directory PH142_Fall2019 (Ph142 is the abbreviation for the course "Introduction to Probability and Statistics in Biology and Public Health").
+
+![alt text](user1_snapshot.png "Single Rsession file whose size is 785 GB")
+
+We paid additional cloud storage costs amounting to few TB of data due to such user actions. Inorder to avoid such scenarios, we need to regularly run jobs to measure and visualize disk size. It will help us to identify and correct user actions before it costs us a lot of $$$.
+
+Policy recommendation is to **not archive any user's home directories whose files size are greater than 100 GB**. Infra team should reach out to the concerned user(s) to inform them about their current storage status and request them to back up their data within 30 days post which the data gets deleted.
+
+## Policy proposal for users home directory archival
+
+As shown in the snapshot below, We had 54 archival requests in total during the course of past two semesters (from Sep, 21 to May, 22). 
+
+![alt text](archival_request.PNG "Archival requests during Fall 2021 and Spring 2022")
+
+99% of the archival requests were made for two different hubs - Datahub and Data 100 hubs. We can hypothesize that most users using other hubs are either a) not aware of the archival service or b) do not require the service. If we plan to improve our outreach message to ensure that other hub users are aware of the archival process, we may still run into issues with our capacity to handle such requests.  We collectively identify that the manual storage archival process is something we want to move away from via automation.
+
+Less than 1% (~50) of all Datahub users (~10k) make data archival requests. To solve this demand from a small subset of users, we allocate a significant amount of storage (as much as 31 TB) as shown in the first snapshot above. Google cloud estimator hypothesizes that it may cost us closer to $5000 per year for storing this volume of data. Here is a [link](https://cloud.google.com/products/calculator/#id=686b9639-ae2e-4a94-a5b9-30aeb1135e6c) to an approximate estimate of cloud costs. 
```

Comment:
Based on current usage, it's costing us about 140$ a month, which is about 1700$ a year - not 5000$. If you look at our billing, logs actually cost us more. 
<img width="1080" alt="image" src="https://user-images.githubusercontent.com/30430/169236232-cab156be-2585-4f26-b0d6-28ecad2d1465.png">

So if the goal is to reduce cost, there are multiple other avenues for us to look at before we start deleting user data. 

I think we can also switch these to archive storage - see https://cloud.google.com/products/calculator/#id=60effa0c-499a-4f0b-b2a2-fb4cfa0276d8 for calculation of how much that would cost for a year, about 280$ for the same 20TB.

Reply:
My suggestion here is that we switch our archived objects to archive class rather than 'standard' class, as that matches our use case for them much better. You can see more about the pricing for different storage classes here: https://cloud.google.com/storage/pricing.

Reply:
I think a fundamental question to me here is - what is the problem we are trying to solve? Is it extra load on @felder in servicing these requests? Or the cost of archiving? If cost is the issue, I think switching it to archival will help give us a 94% reduction in cost (I opened https://github.com/berkeley-dsep-infra/datahub/issues/3389 to track). If workload is the issue, we can find ways to do more automation there.

Reply:
I've also opened https://github.com/berkeley-dsep-infra/datahub/pull/3388 as a draft communication to end users policy, which I think should help a lot with notifications and informing users how they can be good citizens (thanks to Eric Fraser for the idea).